nisystemlink/clients/feeds/_feeds_client.py

```diff
@@ -0,0 +1,190 @@
+from nisystemlink.clients.core._uplink._methods import delete, get, post
+from uplink import Part, Path, Query
+
+from . import models
+
+
+class FeedsClient(BaseClient):
+    def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
+        """Initialize an instance.
+
+        Args:
+            configuration: Defines the web server to connect to and information about
+                how to connect. If not provided, the
+                :class:`HttpConfigurationManager <nisystemlink.clients.core.HttpConfigurationManager>`
+                is used to obtain the configuration.
+
+        Raises:
+            ApiException: if unable to communicate with the Feeds Service.
+        """
+        if configuration is None:
+            configuration = core.HttpConfigurationManager.get_configuration()
+
+        super().__init__(configuration, base_path="/nifeed/v1/")
+
+    @post("feeds")
+    def create_feed(self, feed: models.CreateFeedRequest) -> models.Feed:
+        """Create a new feed with the provided feed details.
+
+        Args:
+            feeds (models.CreateFeedsRequest): Request to create the feed.
+
+        Returns:
+            models.Feed: Details of the created feed.
+
+        Raises:
+            ApiException: if unable to communicate with the Feeds Service.
+        """
+        ...
+
+    @get("feeds", args=[Query, Query])
+    def __query_feeds(
+        self,
+        platform: Optional[str] = None,
+        workspace: Optional[str] = None,
+    ) -> models.QueryFeedsResponse:
+        """Lists available feeds for the Platform `platform` under the Workspace `workspace`.
+
+        Args:
+            platform (Optional[str]): Information about system platform. Defaults to None.
+            workspace (Optional[str]): Workspace id. Defaults to None.
+
+        Returns:
+            models.QueryFeedsResponse: List of feeds.
+
+        Raises:
+            ApiException: if unable to communicate with the Feeds Service.
+        """
+        ...
+
+    def query_feeds(
+        self,
+        platform: Optional[models.Platform] = None,
+        workspace: Optional[str] = None,
+    ) -> List[models.Feed]:
+        """Lists available feeds for the Platform `platform` under the Workspace `workspace`.
+
+        Args:
+            platform (Optional[models.Platform]): Information about system platform. \
+Defaults to None.
+            workspace (Optional[str]): Workspace id. Defaults to None.
+
+        Returns:
+            List[models.Feed]: List of feeds.
+
+        Raises:
+            ApiException: if unable to communicate with the Feeds Service.
+        """
+        platform_by_str = platform.value if platform is not None else None
+        response = self.__query_feeds(
+            platform=platform_by_str,
+            workspace=workspace,
+        ).feeds
+
+        return response
+
+    @post(
+        "feeds/{feedId}/packages",
+        args=[Path(name="feedId"), Query(name="ShouldOverwrite")],
```

Comment:
I believe if you have `args` they needs to match the arguments of the function but this is missing an entry for `package`
```python
args=[Path(name="feedId"), Part(), Query(name="shouldOverwrite")]
```
Or, I think you can drop `args` annotation and type hint the arguments directly since this is private and we're not concerned about keeping this the native python types.
```python
    def __upload_package(
        self,
        feed_id: Path("feedId"),
        package: Part(),
        overwrite: Query("shouldOverwrite") = False,
    ) -> models.Package:
```
You can also add types to Path Part and Query but I don't think those are required.